.docs/index.md

```diff
@@ -119,9 +92,12 @@ libstorage:
   service: virtualbox
 ```
 
-Refer to the
-[VirtualBox documentation](http://libstorage.readthedocs.io/en/stable/user-guide/storage-providers/#virtualbox)
-for additional VirtualBox configuration options.
+Refer to the [VirtualBox documentation](http://libstorage.readthedocs.io/en/stable/user-guide/storage-providers/#virtualbox) for additional VirtualBox configuration options.
```

Comment:
i think i fixed the 80-character count. please let me know if i did it correctly.